### ISSUE TITLE
Widget slows down the dashboard a lot when there are many logs.

### ISSUE THREAD

**Issue by julien-gargot** [louis-ev/kirby-tracking#21]
We should create a cashed version or something…

**Comment by louis-ev** [comment]
The monthly logging system was designed for this, but for high-traffic websites it isn’t enough, yes.
Should we group visits by week or even day? Set this as an option in config.php?
`c::set('kirbytracking.groupby', 'day');` ?

**Comment by julien-gargot** [comment]
Yes, it’s a good idea. Also maybe could we do it automatically after a certain amount of users per day?

But, I believe it will not fix the issue with the dashboard… for that using Ajax to get logs after the dashboard is load could be a good solution.

**Comment by Wemago** [comment]
I would do something like this
```
Tracking
- Year
-- Month
--- Day
---- IP Username file
```

This way we can use Kirby's functions to calculate stats per year, month, day and even user in each tracking sub page, we could even show some graphs using snippetfield plugin.
